.github/workflows/validate.yml

```
@@ -125,3 +125,23 @@ jobs:
       run: |
         . /etc/profile.d/lmod.sh
         tests/ci/run_build.py ohpc ${{ steps.files.outputs.added_modified }}
+
+  build_on_centos_stream_9:
+    runs-on: ubuntu-latest
+    name: Build on CentOS Stream 9
+    container:
+      image: quay.io/centos/centos:stream9
+    steps:
+    - name: Install git
+      run: dnf -y install git
+    - uses: actions/checkout@v2
```

v3 is [available](https://github.com/actions/checkout/tree/v3)
```suggestion
    - uses: actions/checkout@v3
```

---

changed